src/Event/Http/HttpRequestEvent.php

```diff
@@ -20,15 +20,28 @@ final class HttpRequestEvent implements LambdaEvent
     private $headers;
     /** @var string */
     private $queryString;
+    /** @var float */
+    private $payloadVersion;
 
     public function __construct(array $event)
     {
-        if (! is_array($event) || ! isset($event['httpMethod'])) {
+        if (! is_array($event)) {
+            throw new InvalidLambdaEvent('API Gateway or ALB', $event);
+        }
```

Comment:
I don't think this is necessary - the type hint on the constructor should prevent this from ever occurring.

Reply:
Good call - fixed :)